### DISCUSSION TITLE
Error message with the new update

### DISCUSSION THREAD

**Discussion by JamesBiboops** [vuejs/devtools-v6#1681] [Q&A]
Hello,
This morning, I have an error message while using Devtools and I realize that there was an update overnight:
Uncaught (in promise) TypeError: options.version is undefined
backend.js:949:32
Could you tell me what I need to do to resolve this situation.
Thank you

**Comment by Akryum** [comment]
Please open an issue with a reproduction so we can look into it. Thanks :pray:
